tasks/issue.py

```diff
@@ -93,7 +93,7 @@ def ask_reviews(_, pr_id):
                 (chan for team, chan in GITHUB_SLACK_REVIEW_MAP.items() if team.casefold() == reviewer.casefold()),
                 HELP_SLACK_CHANNEL,
             )
-            message = f'Hello :{random.choice(WAVES)}:!\n*{actor}* would like you to review <{pr.html_url}/s|{pr.title}>?\nThanks in advance!'
+            message = f'Hello :{random.choice(WAVES)}:!\n*{actor}* is asking review for PR <{pr.html_url}/s|{pr.title}>.\nCan you please have a look?\nThanks in advance!'
```

Comment:
I think it's better like this but my english isn't perfect 😅 
```suggestion
            message = f'Hello :{random.choice(WAVES)}:!\n*{actor}* is asking review for PR <{pr.html_url}/s|{pr.title}>.\nCould you please have a look?\nThanks in advance!'
```